src/django/api/models.py

```diff
@@ -1,3 +1,109 @@
 from django.db import models
+from django.contrib.gis.db import models
```

Comment:
For clarity, can you change this `models` import to be `as gis_models` or `as geo_models` to differentiate it from the vanilla `models`

If I understand correctly, you also could delete line 1 since the `models` object from the GIS library are extensions of the vanilla `models` object. I think line 2 is overwriting the variable `models` the way this is written anyway.

Reply:
Ok, I qualified the gis models, but kept everything else using django.db.models as the base class. Is this ok, or would it be better for everything to extend the gis.db.models version?